test/controllers/test_categories_controller.rb

```diff
@@ -117,4 +117,27 @@ def test_delete_category
     get "/categories/#{acronym}"
     assert last_response.status == 404
   end
-end
\ No newline at end of file
+
+  def test_parent_category
+    parent_category1 = LinkedData::Models::Category.new(
+      acronym: "PARENT1",
+      name: "Parent Category 1",
+      description: "Description for Parent Category 1."
+    )
+
+    parent_category2 = LinkedData::Models::Category.new(
+      acronym: "PARENT2",
+      name: "Parent Category 2",
+      description: "Description for Parent Category 2."
+    )
+
+    category_instance = LinkedData::Models::Category.new(
+      acronym: "CAT123",
+      name: "Sample Category",
+      description: "This is a sample category.",
+      parentCategory: [parent_category1, parent_category2]
+    )
```

Comment:
here you need to save and fetch the saved instance, to be saved that it is persistent